tiatoolbox/wsicore/wsireader.py

```diff
@@ -4444,3 +4451,509 @@ def read_bounds(
             )
 
         return im_region
+
+
+class AnnotationStoreReader(WSIReader):
```

Comment:
Should it not be defined in annotation package?

Reply:
As it is a subclass of WSIReader, I though the most reasonable place for it to be was in wsireader.py with all the other WSIreader subclassess